Ingestion/templates/python.ecs.template.yaml

```diff
@@ -0,0 +1,356 @@
+  AwsSecretAccessKey:
+    Type: "String"
+    NoEcho: Yes
+    Description: "AWS Secret Access Key, if not specified, will use task execution IAM role"
+  AwsRegion:
+    Type: "String"
+    Description: "AWS Region, only needed if using AwsAccessKeyId and AwsSecretAccessKey"
+  AwsCommandQueueUrl:
+    Type: "String"
+    Default: ""
+    Description: "Command SQS Queue Url, for example: https://sqs.REGION.amazonaws.com/111111111111/xxx"
+  AwsCommandQueueArn:
+    Type: "String"
+    Default: ""
+    Description: "Command SQS Queue ARN, for example: arn:aws:sqs:REGION:11111111111:xxx"
+  ImageTag:
+    Type: "String"
+    Default: ""
+    Description: "specify image tag, will use latest if not specified"
+  TaskCpu:
+    Type: "String"
+    Default: "1024"
+    Description: "The ECS Task CPU value, check https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html"
+    AllowedValues:
+      - 256
+      - 512
+      - 1024
+      - 2048
+      - 4096
+  TaskMemory:
+    Type: "String"
+    Default: "2048"
+    Description: "The ECS Task Memory value (MiB), check https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html"
+    AllowedValues:
+      - 512
+      - 1024
+      - 2048
+      - 4096
+      - 5120
+      - 6144
+      - 7168
+      - 8192
+      - 9216
+      - 10240
+      - 11264
+      - 12288
+      - 13312
+      - 14336
+      - 15360
+      - 16384
+      - 17408
+      - 18432
+      - 19456
+      - 20480
+      - 21504
+      - 22528
+      - 23552
+      - 24576
+      - 25600
+      - 26624
+      - 27648
+      - 28672
+      - 29696
+      - 30720
+Conditions:
+  UseAwsAccessKeyId: !Not [!Equals [!Ref AwsAccessKeyId, ""]]
+  UseAwsSecretAccessKey: !Not [!Equals [!Ref AwsSecretAccessKey, ""]]
+  UseAwsRegion: !Not [!Equals [!Ref AwsRegion, ""]]
+  UseExistingDataHubAccessTokenSecret: !Not [!Equals [!Ref ExistingDataHubAccessTokenSecretArn, ""]]
+  UseDataHubAccessToken: !Not [!Equals [!Ref DataHubAccessToken, ""]]
+  UseImageTag: !Not [!Equals [!Ref ImageTag, ""]]
+
+Resources:
+  DataHubAccessTokenSecret:
+    Type: AWS::SecretsManager::Secret
+    Condition: UseDataHubAccessToken
+    Properties:
+      Name: !Sub "/${AWS::StackName}/DataHubAccessToken"
+      Description: !Sub "${AWS::StackName}: Secrets for DataHubAccessToken"
+      SecretString: !Ref DataHubAccessToken
+  Cluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterName: !Sub "${AWS::StackName}"
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "${AWS::StackName}-log-group"
+  TaskRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub "${AWS::StackName}-task-role"
+      #RoleName: !Sub "datahub-ingestion-task-role"
+      Policies:
+        - PolicyName: ssmmessages-policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+            - Action: 
+                - ssmmessages:CreateControlChannel
```

Comment:
Why are these priveleges required again?

Reply:
needed for fargate login